public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -1446,10 +1446,10 @@ block dart-safe-nav-op
 +makeExample('template-syntax/ts/app/app.component.html', 'safe-1')(format=".")
 :marked
   The view still renders but the displayed value is blank; we see only "The title is" with nothing after it.
-  That is reasonable behavior. At least the app doesn't crash.
+  That is reasonable a behavior. At least the app doesn't crash.
```

Comment:
Again, I commented on this in https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d#commitcomment-18338618